src/renderer/Dataframe.js

```diff
@@ -109,6 +109,24 @@ export default class Dataframe extends DummyDataframe {
         }
     }
 
+    onActive (callback) {
+        this._onActive = callback;
+        if (this.active) {
```

Comment:
I don't know if this makes sense. I added it because a test was failing, but the test itself is testing the implementation and not the API.